resources/js/Molecules/Tablerow.tsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from "react";
+import { Member } from '@/Interfaces/Member';
+
+interface Props {
+    member: Member;
+    handleSelect: (member:Member) => void;
+    type:string;
+};
+
+const Tablerow = ({ member, handleSelect, type }: Props) => {
+    const [fullProfileVisible, setFullProfileVisible] = useState(false);
+
+    const selectedMember = () =>  {
+        handleSelect(member);
+    }
+  
```

Comment:
you could destructure member here : 
const {slack_handle, first_name, last_name, email, mentoring_experience, current_status, interests, specific_interests, timeframe, skillset, suitable_time, previous_experience, mentoring_type, extra_info} = member;

then you can use the properties in the component without the member in front, so mentoring_experience instead of member.mentoring_experience